samples/cancelJob.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(jobId) {
+  // [START bigquery_cancel_job]
+  // Import the Google Cloud client library
+  const {BigQuery} = require('@google-cloud/bigquery');
+
+  async function cancelJob(jobId = 'existing-job-id') {
+    // Attempt to cancel a job.
+
+    // Create a client
+    const bigqueryClient = new BigQuery();
```

Comment:
Let's just call this `client`.  Also - it's super important to instantiate this *outside* of the function, and right after the `require` statement.  It's important because we want users to create this object *once*, and re-use the client.  There's all kind of caching that happens inside of that object, so if they create these in a loop, or ever time a function is called, it's hell on performance.

Reply:
many of the older samples call it bigquery rather than client or bigqueryClient. Does bigquery work or should I transition to client?

Reply:
Honestly either/or is fine by me